src/test/java/com/solvd/DriverPool.java

```diff
@@ -0,0 +1,49 @@
+package com.solvd;
+
+import java.net.MalformedURLException;
+import java.net.URL;
+import java.time.Duration;
+import java.util.ArrayList;
+import java.util.List;
+
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.chrome.ChromeOptions;
+import org.openqa.selenium.remote.RemoteWebDriver;
+
+public class DriverPool {
```

Comment:
Based on the original task, DriverPool is not required.
Just use WebDriver driver in your test